integration/integrate-with-image-registries.adoc

```diff
@@ -0,0 +1,64 @@
+[id="integrate-with-image-registries"]
+= Integrating with image registries
+include::modules/common-attributes.adoc[]
+:context: integrate-with-image-registries
+
+toc::[]
+
+[role="_abstract"]
+{product-title} integrates with a variety of image registries so that you can understand your images and apply security policies for image usage.
+
+When you integrate with image registries, you can view important image details, such as image creation date and Dockerfile details (including image layers).
+
+Once you have integrated with your registry, scan images, view image components, and apply security policies both before you deploy images or for the already deployed images in your clusters.
```

Comment:
Should "both before" be "either before" since there is an "or" between the two options?